public/index.html

```diff
@@ -16,10 +16,31 @@
     <link rel="stylesheet" type="text/css" href="friends.css">
     <link rel="stylesheet" type="text/css" href="editProfile.css">
     <link rel="stylesheet" type="text/css" href="profile.css">
+
+    <!-- script references from websites-->
     <script src="https://cdn.socket.io/socket.io-1.2.0.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
     <script src="https://kit.fontawesome.com/86d78876c2.js" crossorigin="anonymous"></script>
-    <script type="text/javascript" src="client.js"></script>
+
+    <!-- Global variable initliziation -->
+    <script type='text/javascript' > 
```

Comment:
Should we put a comment at the top of each JS file listing these global variables that are available and that they are being initialized in index.html? Just to avoid confusion as to where they are coming from.

Reply:
Good idea. I've made that change